data.sql

```diff
@@ -91,3 +91,235 @@ UPDATE animals
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Pikachu', '01-07-2021', 1, false, 15.04);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Devimon', '05-12-2017', 5, true, 11);
+
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Charmander', '02-08-2020', 0, false, -11), 
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Plantmon', '11-15-2021', 2, true, -5.7), 
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Squirtle', '04-02-1993', 3, false, -12.13), 
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Angemon', '06-12-2005', 1, true, -45), 
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Boarmon', '06-07-2005', 7, true, 20.4), 
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Boarmon', '06-07-2005', 7, true, 20.4), 
+
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Ditto', '05-14-2022', 4, true, 22);
+-- Owners table
+
+INSERT INTO owners (full_name, age)
+VALUES ('Sam Smith', 34);
+
+INSERT INTO owners (full_name, age)
+VALUES ('Jennifer Orwell', 19);
+
+INSERT INTO owners (full_name, age)
+VALUES ('Bob', 45);
+
+INSERT INTO owners (full_name, age)
+VALUES ('Melody Pond', 77);
+
+INSERT INTO owners (full_name, age)
+VALUES ('Dean Winchester', 14);
+
+INSERT INTO owners (full_name, age)
+VALUES ('Jodie Whittaker', 38);
+
+-- Specie table
+INSERT INTO species (name)
+VALUES ('Pokemon');
+
+INSERT INTO species (name)
+VALUES ('Digimon');
+
+-- UPdate species_id
+UPDATE animals
+SET species_id = 2
+WHERE name LIKE '%mon%';
+
+
+UPDATE animals
+SET species_id = 1
+WHERE species_id IS NULL;
+
+-- Update owners_id
+
+UPDATE animals
+  SET owner_id = (SELECT id from owners WHERE full_name = 'Sam Smith')
+  WHERE name = 'Agumon';
+
+UPDATE animals
+  SET owner_id = (SELECT id from owners WHERE full_name = 'Jennifer Orwell')
+  WHERE name IN ('Gabumon', 'Pikachu');
+
+UPDATE animals
+  SET owner_id = (SELECT id from owners WHERE full_name = 'Bob')
+  WHERE name IN ('Devimon', 'Plantmon');
+
+UPDATE animals
+  SET owner_id = (SELECT id from owners WHERE full_name = 'Melody Pond')
+  WHERE name IN ('Charmander', 'Squirtle', 'Blossom');
+
+UPDATE animals
+  SET owner_id = (SELECT id from owners WHERE full_name = 'Dean Winchester')
+  WHERE name IN ('Angemon', 'Boarmon');
+-- Insert the following data for visits:
+INSERT INTO visits (vet_id, animal_id, date)
+VALUES(
+  (SELECT id FROM vets WHERE name = 'William Tatcher' LIMIT 1),
+  (SELECT id FROM animals WHERE name = 'Agumon' LIMIT 1),
+  'May 24, 2020'
+);
+
+INSERT INTO visits (vet_id, animal_id, date)
+VALUES(
+  (SELECT id FROM vets WHERE name = 'Stephanie Mendez' LIMIT 1),
```

Comment:
* I get an error when I try to run these queries reason because you have not added any data to your `vets` table hence no `vet_id` value is recorded, below 👇 is a screenshot of the said error. Kindly fix it 🙏 
![image](https://user-images.githubusercontent.com/52497246/212860793-2fec38fb-4d93-4004-8347-21be6f826f6b.png)